tests/cases/compiler/checkJsFiles.ts

```diff
@@ -0,0 +1,6 @@
+// @allowJs: true
+// @checkJsFiles: true

```

Comment:
You might want to add `@noEmit: true` or `@outFile` to suppress all the `Cannot write file... because it would overwrite` errors in the baselines.